src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for producing a printable string representation of the object
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
Header comment missing here. Use `Returns...` as a description and omit `@return` line.